### DISCUSSION TITLE
ignore specific devices in the statistic

### DISCUSSION THREAD

**Discussion by zadigre** [AdguardTeam/AdGuardHome#2969] [Ideas]
hello everyone!

is there a way to ignore specific devices in the statistics?

I'm asking the question because my Raspberry PI is going through AdGuard Home for the dns. I would like to ignore request coming from my RPI in the statistics count. 
I could configure my RPI to use an other DNS... but this would break other things that relies on reverse DNS.

feature request or is there a way to do it already?

**Comment by ameshkov** [comment]
Nope, not really. I think there was a feature request about that, though.
